testfixtures/tests/test_logcapture.py

```diff
@@ -25,6 +26,58 @@ def test_simple(self):
         root.info('after')
         assert str(l) == "root INFO\n  during"
 
+    def test_simple_strict(self):
+        log_capture = LogCapture(ensure_checks_above=ERROR)
+        root.error('during')
+        log_capture.uninstall()
+        with ShouldAssert("Not asserted ERROR log(s): [('root', 'ERROR', 'during')]"):
+            log_capture.ensure_checked()
+
+    def test_simple_strict_patched(self):
+        re_defaulted_init = partialmethod(LogCapture.__init__, ensure_checks_above=ERROR)
```

Comment:
Having the class attribute with the default level in would allow your monkeypatching to be significantly more pleasant :-)